forge/routes/auth/oauth.js

```diff
@@ -71,7 +71,7 @@ module.exports = async function (app) {
             if (!authClient) {
                 return badRequest(reply, 'invalid_request', 'Invalid client_id')
             }
-            if (!/^editor($|-)/.test(scope)) {
+            if (!/^(editor($|-))|httpAuth-/.test(scope)) {
```

Comment:
I'd be tempted to extract this regex to something like the below to make it clearer what the test is checking for!

`const isNodeReadHeaderRegex = `/^(editor($|-))|httpAuth-/``